audio/miles_adlib.cpp

```diff
@@ -723,7 +726,7 @@ void MidiDriver_Miles_AdLib::updatePhysicalFmVoice(byte virtualFmVoice, bool key
 
 	if (registerUpdateFlags & kMilesAdLibUpdateFlags_Reg_40) {
 		// Calculate new volume
-		byte midiVolume = _midiChannels[midiChannel].currentVolume;
+		byte midiVolume = (_midiChannels[midiChannel].currentVolume * _masterVolume) >> 8;
```

Comment:
Why are you changing here the behaviour of the driver? The rest of the driver users are not aware of this change, and still pass volume as a byte, and will effectively mute sound now.

Reply:
AFAIK all other games handle volume control outside of this module. We defaulting this value to 255, so other games should work fine.